exercises/concept/tisbury-treasure-hunt/.docs/hints.md

```diff
@@ -28,7 +28,7 @@
 - Remember that tuples support all [common sequence operations][common sequence operations].
 - Could you re-use your `compare_records()` function here?
 
-## 5. "Clean up" & format a report of all records
+## 5. "Clean up" & make a report of all records
```

Comment:
Since we are cleaning up a little bit, perhaps the ampersand is a bit "cody".  I would use "Clean up and make a report of all records" instead.

But approving anyway.